gallery/how-to/finding-packages/searching-by-compatibility.md

```diff
@@ -0,0 +1,61 @@
+---
+ms.date:  12/11/2018
+contributor:  JKeithB, SydneyhSmith
+keywords:  gallery,powershell,cmdlet,psgallery
+title:  Packages with compatible PowerShell Editions or Operating System
+---
+# Packages with compatible PowerShell Editions or Operating Systems
+
+Starting with version 5.1, PowerShell is available in different editions which denote varying
+feature sets and platform compatibilities.
+
+## Searching by PowerShell Edition 
+
+- **Desktop Edition:** Built on .NET Framework and provides compatibility with scripts and modules
+  targeting versions of PowerShell running on full footprint editions of Windows such as Server Core
+  and Windows Desktop.
+- **Core Edition:** Built on .NET Core and provides compatibility with scripts and modules
+  targeting versions of PowerShell running on reduced footprint editions of Windows such as Nano
+  Server and Windows IoT.
+
+### PowerShell Gallery extracts supported PSEditions metadata and allows you to filters the packages compatible for specific PowerShell Editions
+
+If a package has compatible PSEditions specified, they will be listed as part of 'PowerShell
+Editions' in the package display page and also in packages results.
+
+![Item display page with PSEditions](../../Images/packagedisplaypagewithpseditions.PNG)
+
+### Search for packages in the gallery UI which works on PowerShellCore
+
+Use Tags:"PSEdition_Desktop" and Tags:"PSEdition_Core" to filters the packages on PowerShell Gallery.
+
+#### Use Tags:"PSEdition_Core" to search items compatible with PowerShell Core Edition.
+
+![Search results for items compatible with Core PSEdition](../../Images/SearchResultsWithPSEditions.PNG)
```

Comment:
Images need to be sized appropriately and need a border. See https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-create-screenshot?branch=master for more information about screenshots.

Recommended dimensions no larger than 1200 px wide.